src/lib/tutorials/normalize-product-like-slug.ts

```diff
@@ -15,11 +15,15 @@ import {
  * This is very specifically targeted at normalizing "cloud" to "hcp".
  * In Tutorials contexts, we use "cloud"; in Dev Dot we use "hcp".
  */
-export function normalizeSlugForDevDot(productSlug: string): ProductSlug {
+export function normalizeSlugForDevDot(
+	productSlug: string
+): ProductSlug | SectionOption {
```

Comment:
```suggestion
): ProductSlug | SectionOption | 'cloud' {
```
I think? or is 'cloud' in `ProductSlug`?